core/index.js

```diff
@@ -45,6 +45,14 @@ async function core (options) {
 
     data[name].releases[`v${versionSemver.version}`].semver = semverToReturn
 
+    data[name].releases[`v${versionSemver.version}`].date = versions[version].date ?? undefined
```

Comment:
I've gotten to dig in a bit now after fixing up my personal laptop setup. I do have one request here, would it be possible to name this `release` or `releaseDate` rather than `date`? My goal with this module is specifically to make it so the code can be a readable API without having to refer to docs or having to already have experience with `dist/index.json` or nodejs/release data. As such, IMO given the context you've provided here I think either of those names would be more clear.

```suggestion
    data[name].releases[`v${versionSemver.version}`].date = versions[version].release ?? undefined
```

```suggestion
    data[name].releases[`v${versionSemver.version}`].date = versions[version].releaseDate ?? undefined
```

Reply:
I think it should be:
```js
data[name].releases[`v${versionSemver.version}`].releaseDate = versions[version].date ?? undefined
```
No?